share/cyclus-flat.rng.in

```diff
@@ -158,6 +158,17 @@ datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
     </element>
   </zeroOrMore>
 
+  <zeroOrMore>
+    <element name="transportunit">
```

Comment:
does a `transportunit` have to declare what kinds of `packages` it can hold?

Reply:
I asked that question myself above. Right now, no it does not

Reply:
Doh - I didn't read your comments, sorry!